cmd/app/main.go

```diff
@@ -14,6 +25,38 @@ func main() {
 		log.Fatalf("Config error: %s", err)
 	}
 
-	// Run
+	log := logger.New(cfg.Log.Level)
+
+	customResolver := aws.EndpointResolverWithOptionsFunc(func(service, region string, options ...interface{}) (aws.Endpoint, error) {
+		return aws.Endpoint{
+			PartitionID:       "aws",
+			URL:               cfg.AWS.EndpointUrl,
+			SigningRegion:     cfg.AWS.Region,
+			HostnameImmutable: true,
+		}, nil
+	})
+
+	awsCfg, err := awsconfig.LoadDefaultConfig(
+		context.Background(),
+		awsconfig.WithEndpointResolverWithOptions(customResolver),
+	)
+	if err != nil {
+		panic("configuration error: " + err.Error())
+	}
+
+	sqsClient := sqspkg.NewClient(sqsservice.NewFromConfig(awsCfg), fmt.Sprintf("%s/000000000000/%s", cfg.AWS.EndpointUrl, cfg.AWS.FakeNewsQueueName))
```

Comment:
Mozes ovaj [option](WithMaxNumberOfMessages) da dodas za max number of messages?

Reply:
sorry ignorisi, sad vidim da je to option za Receive 🤦